server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceMetrics.java

```diff
@@ -26,17 +27,28 @@ public record AllocationStats(long unassignedShards, long totalAllocations, long
     public record NodeWeightStats(long shardCount, double diskUsageInBytes, double writeLoad, double nodeWeight) {}
 
     public static final DesiredBalanceMetrics NOOP = new DesiredBalanceMetrics(MeterRegistry.NOOP);
+
     public static final String UNASSIGNED_SHARDS_METRIC_NAME = "es.allocator.desired_balance.shards.unassigned.current";
     public static final String TOTAL_SHARDS_METRIC_NAME = "es.allocator.desired_balance.shards.current";
     public static final String UNDESIRED_ALLOCATION_COUNT_METRIC_NAME = "es.allocator.desired_balance.allocations.undesired.current";
     public static final String UNDESIRED_ALLOCATION_RATIO_METRIC_NAME = "es.allocator.desired_balance.allocations.undesired.ratio";
+
     public static final String DESIRED_BALANCE_NODE_WEIGHT_METRIC_NAME = "es.allocator.desired_balance.allocations.node_weight.current";
     public static final String DESIRED_BALANCE_NODE_SHARD_COUNT_METRIC_NAME =
         "es.allocator.desired_balance.allocations.node_shard_count.current";
     public static final String DESIRED_BALANCE_NODE_WRITE_LOAD_METRIC_NAME =
         "es.allocator.desired_balance.allocations.node_write_load.current";
     public static final String DESIRED_BALANCE_NODE_DISK_USAGE_METRIC_NAME =
         "es.allocator.desired_balance.allocations.node_disk_usage_bytes.current";
+
+    public static final String CURRENT_NODE_SHARD_COUNT_METRIC_NAME = "es.allocator.allocations.node_shard_count.current";
+    public static final String CURRENT_NODE_WRITE_LOAD_METRIC_NAME = "es.allocator.allocations.node_write_load.current";
+    public static final String CURRENT_NODE_DISK_USAGE_METRIC_NAME = "es.allocator.allocations.node_disk_usage_bytes.current";
+    public static final String CURRENT_NODE_UNDESIRED_SHARD_COUNT_METRIC_NAME =
+        "es.allocator.allocations.node_undesired_shard_count.current";
+    public static final String CURRENT_NODE_FORECASTED_DISK_USAGE_METRIC_NAME =
+        "es.allocator.allocations.node_forecasted_disk_usage_bytes.current";
```

Comment:
`NodeAllocationStats` gives these two more compared to the `DESIRED_BALANCE_...` metrics above. Not sure if they're needed, but I have exported all of them.